source/elements/oneMKL/source/domains/rng/device_routines/device-distributions.rst

```diff
@@ -0,0 +1,95 @@
+.. SPDX-FileCopyrightText: 2023 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _onemkl_device_rng_distributions:
+
+Device Distributions
+====================
+
+
+oneMKL RNG routines are used to generate random numbers with different types of distribution. Each function group is 
```

Comment:
What do you mean by `distribution`?
1. A distribution
2. The distribution
3. Distributions

Reply:
I believe there shall be distributions. Changed